### PR TITLE
added delay parameter

### PR DESCRIPTION
**Pull request by gtfisher**
Signed-off-by: Grant TF <grant@tarrant-fisher.co.uk>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,4 +7,5 @@
      Add "get_all_events" to the api module
      Add "convert_time and get_all_events_in_window to probes module
      Add "unit tests for convert time"
+     Add "delay parameter in probe and added usage in readme"
```
Comment (russmiles) [inline CHANGELOG.md]: Changelog should have " -   " for each of the added/deleted/changed mentions.

File: chaosinstana/probes.py [hunk 2]
```diff
@@ -10,12 +11,15 @@
 
 
 def get_all_events_in_window(from_time: str = None, to_time: str = None,
+                             delay: int = 0,
```
Comment (russmiles) [inline chaosinstana/probes.py:14]: Needs to be formatted differently.

File: chaosinstana/probes.py [hunk 3]
```diff
@@ -10,12 +11,15 @@
...
                              configuration: Configuration = None,
                              secrets: Secrets = None) -> str:
     """
     get all events from instana within a time window, given by the from_time
     and the to_time
     """
+    logger.debug("delay is: {}".format(delay))
```
Comment (russmiles) [inline chaosinstana/probes.py]: Might be worth being more specific here? "Instana probe delay is: {}" for example?

**Review by russmiles — CHANGES_REQUESTED** [review]
Very close, some small changes only.

File: chaosinstana/probes.py (hunk 2 above)
Comment (gtfisher) [reply chaosinstana/probes.py:14]: Is correct with the linter

File: CHANGELOG.md (hunk 1 above)
Comment (gtfisher) [reply CHANGELOG.md]: done

**Pull request merged**